code/modules/swarmers/swarmer_act.dm

```diff
@@ -5,6 +5,29 @@
   * Arguments:
   * * S - A reference to the swarmer doing the interaction
   */
+#define DANGEROUS_DELTA_P 250	//Value in kPa where swarmers arent allowed to break a wall or window with this difference in pressure.
+
+/turf/proc/return_turf_delta_p()	//Finds the greatest difference in pressure across a turf, only considers open turfs.
+	var/pressure_greatest = 0
+	var/pressure_smallest = INFINITY 					//Freaking terrified to use INFINITY, man
+	for(var/t in RANGE_TURFS(2, src))			//Begin processing the delta pressure across the wall.
+		var/turf/open/turf_adjacent = t
+		if(!istype(turf_adjacent))
+			continue
+		pressure_greatest = max(pressure_greatest, turf_adjacent.air.return_pressure())
+		pressure_smallest = min(pressure_smallest, turf_adjacent.air.return_pressure())
+
+	return pressure_greatest - pressure_smallest
+
+/turf/proc/is_nearby_planetary_atmos()	//Runs through all adjacent open turfs and checks if any are planetary_atmos returns true if even one passes
+	. = FALSE
+	for(var/t in RANGE_TURFS(2, src))
```

Comment:
```suggestion
	for(var/t in RANGE_TURFS(1, src))
```
2 will make a 5x5 box
1 will make a 3x3
I assume you want the latter?

Reply:
Huh, I was told that 1 only got you the turf below.

Reply:
```
#define RANGE_TURFS(RADIUS, CENTER) \
  block( \
    locate(max(CENTER.x-(RADIUS),1),          max(CENTER.y-(RADIUS),1),          CENTER.z), \
    locate(min(CENTER.x+(RADIUS),world.maxx), min(CENTER.y+(RADIUS),world.maxy), CENTER.z) \
  )
```
There's no need to speculate when you can search the definition.